additional_information/cmd/main.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"bootcampProject/additional_information/internal/repository"
+	"bootcampProject/additional_information/internal/service"
+	"bootcampProject/additional_information/internal/transport"
+	pb "bootcampProject/additional_information/proto"
+	"bootcampProject/database"
+	"flag"
+	"fmt"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"google.golang.org/grpc"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+const (
+	defaultGrpcPort = ":9080"
```

Comment:
you should get this from env vars in a config file with Config struct, and have default values like:
`type Config struct {
	// Database
	DBHost     string `envconfig:"DB_HOST" default:"localhost"`
	DBUser     string `envconfig:"DB_USER"`
	DBPassword string `envconfig:"DB_PASSWORD"`
	DBName     string `envconfig:"DB_NAME"`
	DBPort     string `envconfig:"DB_PORT" default:"5432"`
}`

Reply:
I implemented YAML file with "envconfig" library